README.md

```diff
@@ -40,34 +40,31 @@ If the SFTP service is used, the question arises regarding how the system will k
 
 1. Clone this repository.
 2. Change directories into the `base-local` directory.
-3. Determine whether to store encrypted files on the local file system or in AWS S3 (*hint: for this initial version, choose filesystem*).
-4. Copy the proper files based on the storage selection; for example:
+3. Adjust if needed values in [.env file](./env)
```

Comment:
Since the .env file will have secrets included, it needs to be protected from accidental inclusion in a commit. So the existing `.env` file should be renamed `.env.sample` (consistent with my practice elsewhere, or perhaps `.env.default`). Also, the filename `.env` should be included in the repo's `.gitignore` file. Last include a line in the set of cp commands to copy the sample to the working `.env` file. (And then put this instruction step after the cp commands.)

Reply:
I renamed `.env` to `.env.default`, added `.env` to `.gitignore` and added additional instructions to `README.md`